beamer/tests/agent/utils.py

```diff
@@ -46,14 +47,15 @@ def generate_artifacts(artifacts_dir, contracts):
             },
         },
     }
-    artifacts_dir.mkdir()
-    with artifacts_dir.joinpath("1337-ethereum.deployment.json").open("wt") as f:
+    artifacts_dir.mkdir(exist_ok=True)
+    path = artifacts_dir / f"{chain_id}-ethereum.deployment.json"
```

Comment:
> not sure about this part, what if chain id changes because of an upgrade or something then we dont have config json at all.

This is not about config (artifacts), here we are generating the deployment artifacts.
Changes in chain ID should not affect anything here since we don't look up those, at least in our automated tests.
(The E2E test and container image test explicitly mention the chain ID, but those are fine and are not related to this.)

However, your comment made me look at `beamer/tests/config/util.py`, and there we could/should do the same thing as we are doing here. No reason to hardcode chain IDs.

Reply:
I've now removed hardcoded chain IDs there.